src/features/MovieList/MovieTile/styled.js

```diff
@@ -1,125 +1,158 @@
         width: 50%;
     }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        width: 40%;
+    }
 `;
 
 export const Wrapper = styled.div`
     display: flex;
     flex-direction: column;
     height: 100%;
-    justify-content: space-between;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         margin-left: 16px;
-        width: 50%;
-        display: flex;
-        justify-content: flex-start;
     }
 `;
 
-export const DescriptionContainer = styled.div`
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
-          
-        }
-`;
-
 export const Title = styled.h3`
-    margin: 16px 0px 8px;
+    margin: 16px 0 8px;
     font-weight: 500;
     font-size: 22px;
     line-height: 130%;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px) {
         font-size: 16px;
+        margin: 0 0 4px;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        font-size: 4vw;
     }
 `;
 
-export const Year = styled.p`
+export const Year = styled.span`
     margin: 0;
     font-size: 16px;
     line-height: 150%;
     color: ${({ theme }) => theme.color.darkerGray};
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         font-size: 13px;
-        line-height: 130%;
+    }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        font-size: 2.5vw;
     }
 `;
 
-export const TagsContainer = styled.div`
-    margin-top: 8px;
+export const TagsContainer = styled.ul`
+    margin: 8px 0 0;
+    padding: 0;
+    gap: 8px;
     display: flex;
     flex-wrap: wrap;
-    gap: 8px;
+    list-style: none;
+    margin-bottom: 0;
 `;
 
-export const Tag = styled.p`
-    margin: 0px;
+export const Tag = styled.li`
+    margin: 0;
     padding: 8px 16px;
     background-color: ${({ theme }) => theme.color.gray};
     border-radius: 5px;
     font-size: 14px;
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         padding: 4px 8px;
         font-size: 10px;
     }
+
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px){
+        padding: 1vw 2vw;
+        font-size: 2vw;
+    }
 `;
 
 export const RatingContainer = styled.div`
-    margin-top: 39px;
+    margin-top: 12px;
     display: flex;
+    flex-wrap: wrap;
     align-items: center;
     align-content: flex-end;
     gap: 12px;
-
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+    
+    @media (max-width: ${({ theme }) => theme.breakpoint.tablet}px){
         gap: 8px;
-        margin-top: 12px;
+    }
+
+    @media (min-width: ${({ theme }) => theme.breakpoint.tablet}px){
+        flex: 1;
```

Comment:
You dont need two media queries here. Just set `flex: 1;` for every media and then `flex: 0;` in `@media (max-width: ${({ theme }) => theme.breakpoint.tablet}px)`

Reply:
ok thanks, fixed